src/Components/Form/Form.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import ApiRequest from '../ApiRequest/ApiRequest';
+import './Form.scss';
+
+class Form extends Component {
+  submitForm = event => {
+    event.preventDefault();
+
+    const locationFieldValue = event.target.location.value;
+    const data = ApiRequest(locationFieldValue);
```

Comment:
this is not a DATA. 
This is and endpoint URL from where you will fetch the data. 
Wrong naming.

Refactor this as I mentionded in above comments.
then just simply import that API_ENDPOINT  here to the file and use as 

`fetch(OPEN_WEAHTERJ_API_ENDPOINT)`

data will come from fetch.
Now under data const you have some URL. So is the URL 'www.www.upda.pl' a data for you??